target/linux/ramips/image/mt7621.mk

```diff
@@ -470,6 +470,20 @@ define Device/bolt_arion
 endef
 TARGET_DEVICES += bolt_arion
 
+define Device/comfast_cf-e390ax
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 15808k
+  DEVICE_VENDOR := ComFast
+  DEVICE_MODEL := CF-E390AX
+  DEVICE_PACKAGES := kmod-mt7915-firmware kmod-mt7916-firmware kmod-mt7915e -uboot-envtools
```

Comment:
`kmod-mt7915e`  is not needed here, `mt7915-firmware`  and `mt7916-firmware` already select it.

Is `kmod-mt7916-firmware` really needed here? do you need the files provided by it?

Reply:
Fixed